commands/plugins/bs.go

```diff
@@ -0,0 +1,42 @@
+package plugins
+
+import (
+	"fmt"
+	"math/rand"
+	"regexp"
+	"time"
+
+	"github.com/andreyrd/bogon/commands"
+)
+
+var bsCommand = func() (string, *regexp.Regexp, commands.CommandFunc, commands.Options) {
+	out := regexp.MustCompile("(?i)^\\.bs$")
+	return bsTitle, out, bsGenerate, defaultOptions
+}
+
+var adverbs = []string{"abstractly", "analytically", "disruptively", "efficiently", "financially", "holistically",
+	"sufficiently", "sustainably", "quantifiably", "quickly"}
+var verbs = []string{"analyzing", "encapsulating", "disrupting", "envisioning", "growing", "leveraging", "redefining",
+	"reenvisioning", "sustaining", "testing", "unboxing"}
+var nouns = []string{"alignment", "boundaries", "clutter", "density", "evolution", "horizons", "logistics", "missions",
+	"paradigms", "potential", "sustainability", "transformation", "trust"}
+
+func bsGenerate(msg commands.Message, ret commands.MessageFunc) string {
+	// Choose a random adverb
+	rand.Seed(time.Now().UTC().UnixNano())
```

Comment:
Ah true! I just now realized that will UnixNano() will never ever return the same number, so I don't have to worry about getting the same sequence from the same seed. Fixed!